enterprise/models.py

```diff
@@ -223,41 +238,44 @@ class Meta:
     )
 
     enable_data_sharing_consent = models.BooleanField(
+        "Activate data sharing consent prompt",
```

Comment:
it wasn't clear to me that this param is intended as the `verbose_name`, could we add the named param to this and the other fields below that include verbose names?